imagersite/imager_profile/tests.py

```diff
@@ -60,3 +80,46 @@ def test_delete_profile_deletes_user(self):
         self.assertEqual(len(ImagerProfile.objects.all()), 1)
         user.profile.delete()
         self.assertEqual(len(User.objects.all()), 0)
+
+
+class ProfileViewTestCase(TestCase):
+    def setUp(self):
+        user = UserFactory(username='userbob', email='bob@example.com')
+        user.set_password('secret')
+        user.save()
+        user.profile.camera = 'Canon'
+        user.profile.address = '123 Bob Drive'
+        user.profile.website = 'www.bob.com'
+        user.profile.photography_type = 'portraits'
+        PhotoFactory(title='Bobs Photo', user=user.profile)
+        AlbumFactory(title='Bobs Album', user=user.profile)
+        user.profile.save()
+
+        user2 = UserFactory(username='usereve', email='eve@example.com')
+        user2.set_password('secret')
+        user2.save()
+
+    def tearDown(self):
+        User.objects.all().delete()
```

Comment:
teardown should not be required, as the test isolation ensures that the changes made to the database are rolled back between tests.